client_internal_test.go

```diff
@@ -24,6 +24,8 @@ func TestClient_NewRequest(t *testing.T) {
 	} else if req.URL.Path != "/v2/accounts/14579" {
 		t.Fatalf("unexpected path: %s", req.URL.Path)
 	} else if req.Method != "GET" {
+	} else if req.Header.Get("Authorization") != "Basic abc" {
```

Comment:
Could you move the `t.Fatalf`'s around -- they are both currently grouped together and the two `else if`s are grouped together.